tasks/managed/prepare-fbc-release/prepare-fbc-release.yaml

```diff
@@ -84,13 +84,27 @@ spec:
             containerImage=$(jq -r '.containerImage' <<< "$component")
             componentName=$(jq -r '.name' <<< "$component")
 
-            # Extract OCP version from container image using Skopeo.
-            # This command retrieves the 'org.opencontainers.image.base.name' annotation,
-            # which includes the OCP version, formatted as "registry:version".
-            # Example: "registry.redhat.io/openshift4/ose-operator-registry:v4.12"
-            # The script then isolates the version part (e.g., "v4.12") from this string.
```

Comment:
I think this would still be useful to have, no? Otherwise it's not clear what we expect to see in the `org.opencontainers.image.base.name` annotation.

Reply:
I probably removed by mistake. I will re-add.